src/test/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobContainerClientBuilderProviderTest.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.blobrouter.services.storage;
+
+import com.azure.core.http.HttpClient;
+import com.azure.storage.blob.BlobContainerClientBuilder;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class BlobContainerClientBuilderProviderTest {
+
+
+    @Mock
+    private HttpClient httpClient;
+
+    private BlobContainerClientBuilderProvider blobContainerClientBuilderProvider
+        = new BlobContainerClientBuilderProvider(httpClient, "http://example.com");
+
+    @Test
+    void should_provide_builder() {
+
+        BlobContainerClientBuilder blobContainerClientBuilder = blobContainerClientBuilderProvider
+            .getBlobContainerClientBuilder();
+
+        assertThat(blobContainerClientBuilder).isNotNull();
```

Comment:
Is `httpClient` and `url` accessible from the client? Might be useful for assertions, if possible.

Reply:
not accessible